Pipeline release failure on Push commit and tag task

### Prerequisites

- [x] Write a descriptive title.
- [x] Make sure you are able to reproduce it on the latest released version
- [x] Search the existing issues.

### Steps to reproduce

Pipeline failing to release

### Expected behavior

```console
It releases
```

### Actual behavior

```console
It fails at the Push commit and tag
```

### Error details

```console
Push commit and tag
0s
1
Run git push --follow-tags origin main
4
error: src refspec main does not match any
5
error: failed to push some refs to 'https://github.com/tonylea/PSSAToJunit'
6
Error: Process completed with exit code 1.
```

### Environment data

```powershell
NA
```

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -129,7 +129,7 @@ jobs:
 
       - name: Push commit and tag
         if: steps.check.outputs.should_release == 'true'
-        run: git push --follow-tags origin main
+        run: git push --follow-tags origin HEAD:main
 
       - name: Create staging folder
         if: steps.check.outputs.should_release == 'true'

